test/passUnwrappedFunctionsTransformer.test.ts

```diff
@@ -1,4 +1,4 @@
-import { transformCode } from './inlineWrappedFunctions.test';
```

Comment:
Importing another test file forced Jest to run the tests inside `inlineWrappedFunctions`, even if you only wanted to run tests for `passUnwrappedFunctionsTransformer`. Also caused the tests from `inlineWrappedFunctions` to be run multiple times (in both test suites).